source/how-tos/debug/debug-apache.rst

```diff
@@ -77,7 +79,50 @@ Or you're using the wrong hostname in your browser.
 
           sudo /sbin/apache2 -S
 
+Performance Tuning
+------------------
+
+If you're servicing many clients at time (more than 50) you will likely need to change the
+`Apache Httpd's MPM configuration`_. The default configuration may degrade service when
+Httpd has to serve many clients (I.e., when you have a lot of customers using Open OnDemand).
+
+We suggest configurations similar to this. 
+
+.. note::
+  The most important directives used to control this MPM are `ThreadsPerChild`, which controls 
+  the number of threads deployed by each child process and `MaxRequestWorkers`, which controls
+  the maximum total number of threads that may be launched.
+  
+  MaxRequestWorkers * ThreadsPerChild will be your effictive cap on the number of 
+  simultaneous requests that will be served.
```

Comment:
Pulled the first bit from apache docs itself. I believe the 2nd line is true. That seems like the starting point. This _could_ be sufficient, but I feel like we need a table or something as a decent starting point? 

Or just say `512` is likely to work well on most systems? and if you have a particularly small system (less than what, 2 cores 4 GB?) maybe `256`?

Reply:
The number I came up with was a combination of cores on the system and max expected concurrent traffic. Once the worker threads are exhausted, people won't be able to access OnDemand so it's important to keep the number well above the expected max usage.

Reply:
Well most folks are going to copy paste what we have I'm sure, so I guess I'm asking if 512 will work on most systems if most systems have more than 8 cores (which I have no idea if that's true, most could have 4).

Reply:
It's better to have fewer ServerLimit and more threads, the server limit is number of processes I believe so that more tied to cores where as threads are less dependent on number of cores.